admin/js/boldgrid-backup-admin-table-includes.js

```diff
@@ -16,7 +16,7 @@ BoldGrid.TableInclude = function( $ ) {
 	'use strict';
 
 	var self = this,
-		$container = $( '#table_inclusion' ),
+		$container = $( 'div#table_inclusion' ),
```

Comment:
All the changes you made to boldgrid-backup-admin-folder-exclude.js where made to this file as well. Please see those comments.

Reply:
See the above comments